frontend/src/Account/UpdateProfile/ProfileDetails.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  Divider,
+  Grid,
+  TextField,
+  makeStyles
+} from '@material-ui/core';
+
+const states = [
+  {
+    value: 'Caifornia',
+    label: 'California'
+  },
+  {
+    value: 'new-york',
+    label: 'New York'
+  },
+  {
+    value: 'Boston',
+    label: 'Boston'
+  }
+];
+
+const useStyles = makeStyles(() => ({
+  root: {}
+}));
+
+const ProfileDetails = ({ className, ...rest }) => {
+  const classes = useStyles();
+  const [values, setValues] = useState({
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    state: '',
+    country: ''
+  });
+
+  const handleChange = (event) => {
+    setValues({
+      ...values,
+      [event.target.name]: event.target.value
+    });
+  };
+
+  return (
+    <form
+      autoComplete="off"
+      noValidate
+      className={clsx(classes.root, className)}
+      {...rest}
+    >
+      <Card>
+        <CardHeader
+          subheader="The information can be edited"
+          title="Profile"
+        />
+        <Divider />
+        <CardContent>
+          <Grid
+            container
+            spacing={3}
+          >
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                helperText="Please specify the first name"
+                label="First name"
+                name="firstName"
+                onChange={handleChange}
+                required
+                value={values.firstName}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Last name"
+                name="lastName"
+                onChange={handleChange}
+                required
+                value={values.lastName}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Email Address"
+                name="email"
+                onChange={handleChange}
+                required
+                value={values.email}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Phone Number"
+                name="phone"
+                onChange={handleChange}
+                type="number"
```

Comment:
This type should be 'tel' instead of number. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel

```suggestion
                type="tel"
```